src/views/AddTenant/addTenant.js

```diff
@@ -168,6 +193,12 @@ export const AddTenant = () => {
 
             setSubmitting(true);
             handleFormSubmit(toSubmit);
+            resetForm();
+            setPropertySearchText("");
+            setPropertySelection([])
+            setStaffSearchText("");
+            setStaffSelections([]);
+            setIsValidationActive(false);
```

Comment:
Wow this seems painful, I think I understand what it is doing. I find it weird that it is necessary. Perhaps one day I will learn React and have opinions about this :-)